Formula/dcmtk.rb

```diff
@@ -17,20 +17,23 @@ class Dcmtk < Formula
 
   option "with-docs", "Install development libraries/headers and HTML docs"
   option "with-libiconv", "Build with brewed libiconv. Dcmtk and system libiconv can have problems with utf8."
+  option "with-dicomdict", "Build with baked-in DICOM data dictionary."
+  option "without-openssl", "Build without support for OpenSSL."
```

Comment:
Thanks for the clarification -- I was just using the doxygen example as a template. Should I just change the "depends_on" line as suggested, or should I also remove the "without-openssl" option as well?